deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -1369,6 +1369,27 @@ Example Usage:
 {{ $enabled }}
 {{- end -}}
 
+{{/*
+Check if otelcol logs collector is enabled.
+It's enabled if both logs in general and the collector specifically are enabled.
+If both the collector and Fluent-Bit are enabled, we error.
```

Comment:
Do we really want to explicitly forbid the combination of both OT logs collector and Fluent Bit?

I can imagine a user might e.g. want to collect container logs with OT and systemd logs with FB (or the other way around) for some reason and I'm not convinced that hardcoding the lack of this possibility is a good idea.

Reply:
I think that customers more likely forget to disable one of the collector during migration than will try to use both of them. Especially when we deprecate fluent-bit

Reply:
I think the value of catching accidental misconfigurations is greater than the value of allowing users to have both simultaneously. Now that OT supports systemd logs, I don't think there's any reason to run them side-by-side.

Reply:
I added it as `allowSideBySide`, along with a documentation section.

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/2436/commits/b7f0f80e64ce8de261f5b0417223fab4ca4038c0